src/90-ceph-osd-sysctl.conf

```diff
@@ -0,0 +1 @@
+fs.aio-max-nr = 1048576
```

Comment:
could you consider using `90-ceph-osd.conf` as the filename? as sysctl is redundant here.

Reply:
OK，I will change the name,thank you!